st2common/st2common/models/system/action.py

```diff
@@ -85,6 +84,20 @@ def _get_command_string(self, cmd, args):
         result = '%s %s' % (cmd, args)
         return result
 
+    def _get_error_result(self, e):
+        """
+        Prepares a structured error result based on the exception.
+
+        :type e: ``Exception``
+
+        :rtype: ``dict``
+        """
+        result = {}
+        result['failed'] = True
+        result['succeeded'] = False
+        result['exception'] = str(e)
```

Comment:
I think that putting the traceback in the result makes it hard to read the result. Also, less human friendly. Thoughts?